namespace.go

```diff
@@ -282,7 +282,7 @@ func (ns *Namespace) negotiateClaim(ctx context.Context, client *amqp.Client, en
 						tab.For(refreshCtx).Error(err)
 						select {
 						case <-refreshCtx.Done():
-							break
+							return
```

Comment:
Hm...makes sense. I guess the claim negotiation would just never end?

Reply:
Yep, stuck in the for loop.